src/main/java/de/srendi/advancedperipherals/common/items/MemoryCardItem.java

```diff
@@ -30,9 +30,8 @@ public boolean isEnabled() {
     public void appendHoverText(ItemStack stack, @Nullable Level levelIn, List<Component> tooltip, TooltipFlag flagIn) {
         super.appendHoverText(stack, levelIn, tooltip, flagIn);
         CompoundTag data = stack.getOrCreateTag();
-        // TODO <0.8>: remove the owner name field
-        if (data.contains("ownerId") && data.contains("owner")) {
-            tooltip.add(EnumColor.buildTextComponent(Component.translatable("item.advancedperipherals.tooltip.memory_card.bound", data.getString("owner"))));
+        if (data.contains("ownerId")) {
+            tooltip.add(EnumColor.buildTextComponent(Component.translatable("item.advancedperipherals.tooltip.binding.boundto", data.getString("ownerId"))));
```

Comment:
alr we are not showing ownerId here, we still want show the owner's name, so we need an extra step to turn the id into `GameProfile`, then get the player's cached name

Reply:
ah yeah
I can use the new cache thing there I made